powertools-parameters/src/test/java/software/amazon/lambda/powertools/parameters/DynamoDbProviderE2ETest.java

```diff
@@ -0,0 +1,98 @@
+package software.amazon.lambda.powertools.parameters;
+
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import software.amazon.awssdk.http.urlconnection.UrlConnectionHttpClient;
+import software.amazon.awssdk.services.dynamodb.DynamoDbClient;
+import software.amazon.awssdk.services.dynamodb.model.AttributeValue;
+import software.amazon.awssdk.services.dynamodb.model.PutItemRequest;
+import software.amazon.lambda.powertools.parameters.cache.CacheManager;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * This class provides simple end-to-end style testing of the DynamoDBProvider class.
+ * It is ignored, for now, as it requires AWS access and that's not yet run as part
+ * of our unit test suite in the cloud.
+ *
+ * The test is kept here for 1/ local development and 2/ in preparation for future
+ * E2E tests running in the cloud CI.
+ */
+@Disabled
+public class DynamoDbProviderE2ETest {
```

Comment:
Should we have this in the e2e tests module instead? You can look in the idempotency module for tests with DDB: https://github.com/awslabs/aws-lambda-powertools-java/blob/master/powertools-idempotency/src/test/java/software/amazon/lambda/powertools/idempotency/persistence/DynamoDBPersistenceStoreTest.java

Reply:
Agreed! As it's not there yet I'd intended to merge it in with this commit so its complete, and then move it over after the E2E one is in master. WDYT @jeromevdl ?

Reply:
OK to keep it here while we merge the e2e in master. But we'll need to add the CDK part to it, to make it autonomous.

Reply:
Agreed - so it'll use the E2E stuff to get itself its own DDB tables for each schema, seed them with data, then test against them - rather than hardcoding some tables that the developer happens to have floating around in their AWS account.